content/en/docs/security/config-best-practices.md

```diff
@@ -0,0 +1,155 @@
+
+- For communication encryption, see
+  [Configuring certificates](/collector/configuration/#setting-up-certificates).
+- For authentication, use the OTel Collector's authentication mechanism, as
+  described in [Authentication](/collector/configuration/#authentication).
+
+## Processors
+
+[Processors](/collector/configuration/#processors) sit between receivers and
+exporters. They are responsible for processing telemetry before it's analyzed.
+From a security perspective, processors are useful in a few ways.
+
+### Recommended processors
+
+<!--- TODO: SHOULD configure recommended processors. If so, what are they?
+
+redaction processor (never tried myself)
+transform processor (great for redacting PIIs and such) -->
+
+### Scrubbing sensitive data
+
+You can use the OpenTelemetry Collector to scrub sensitive data before exporting
+it to a backend. Configure the Collector to obfuscate or scrub sensitive data
+before exporting.
+
+<!--- TODO: SHOULD configure obfuscation/scrubbing of sensitive metadata. How? Give more details and/or link to an existing document -->
+
+Use OpenTelemetry Collector's `redaction` processor to scrub sensitive data.
+
+<!--- TODO: Give example config for the redaction processor or remove this line. --->
+
+### Safeguarding resource utilization
+
+Processors also offer safeguards for resource utilization. After implementing
+safeguards for resource utilization in your
+[hosting infrastructure](/security/hosting-best-practices/), make sure your
+OpenTelemetry Collector configuration uses these safeguards.
+
+<!-- start same page content in hosting-best-practices -->
+
+The `batch` and `memory_limiter` processors help ensure the OpenTelemetry
+Collector is resource efficient and does not run out of memory when overloaded.
+These two processors should be enabled on every defined pipeline.
+
+For more information on recommended processors and how to order them in your
+configuration, see the
+[Collector processor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor)
+documentation.
+
+<!-- /end same page content in hosting-best-practices -->
+
+## Extensions
+
+While receivers, processors, and exporters handle telemetry directly, extensions
+serve different needs.
+
+<!--- TODO: Extensions SHOULD NOT expose sensitive health or telemetry data. How? What can you do? -->
+
+### Health and telemetry
+
+Extensions are available for health check information, Collector metrics and
+traces, and generating and collecting profiling data. When enabled with their
+default settings, all of these extensions, except the health check extension,
+are accessible only locally to the OpenTelemetry Collector. Take care to protect
+sensitive information when configuring these extensions for remote access, as
+they might expose it accidentally.
+
+### Forwarding
+
+A forwarding extension is used when you need to collect telemetry that's not
```

Comment:
I feel like this is misplaced here... I don't think the forwarding extension is related to security, nor is a best practice.

Reply:
Removed in dbfd1b6. Thanks!